source/expressions.tex

```diff
@@ -6938,7 +6938,7 @@
 \item
 an invocation of an implicitly-defined copy/move constructor or
 copy/move assignment operator
-for a union whose active member (if any) is mutable,
+for a union whose active member (if any) is \keyword{mutable},
```

Comment:
We say "const" (no special decoration), so I think we shouldn't make this case a \keyword.

Reply:
The motivation for this was to keep the distinction between objects and members (i,e. a `mutable` member corresponds to a *mutable subobject*).